src/DependencyInjection/HeimrichHannotVideoExtension.php

```diff
@@ -24,22 +24,56 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        if (!isset($config['videoProvider']['youtube'])) {
-            $config['videoProvider']['youtube']['class'] = YouTubeVideo::class;
+        $config = $this->deprecatedOptionsMapping($config);
+
+        if (!isset($config['video_provider']['youtube'])) {
+            $config['video_provider']['youtube']['class'] = YouTubeVideo::class;
         }
 
-        if (!isset($config['videoProvider']['vimeo'])) {
-            $config['videoProvider']['vimeo']['class'] = VimeoVideo::class;
+        if (!isset($config['video_provider']['vimeo'])) {
+            $config['video_provider']['vimeo']['class'] = VimeoVideo::class;
         }
 
-        if (!isset($config['videoProvider']['file'])) {
-            $config['videoProvider']['file']['class'] = FileVideo::class;
+        if (!isset($config['video_provider']['file'])) {
+            $config['video_provider']['file']['class'] = FileVideo::class;
         }
+
+        // Support deperecated option
+        // @todo: remove in version 1.0
+        $config['videoProvider'] = $config['video_provider'];
+
         $container->setParameter('huh_video', $config);
     }
 
     public function getAlias()
     {
         return 'huh_video';
     }
+
+    /**
+     * Support deprecated options.
+     *
+     * @todo Remove in version 1.0
+     */
+    protected function deprecatedOptionsMapping(array $config): array
+    {
+        if (true === $config['enableNewsSupport'] && true !== $config['enable_news_support']) {
+            $config['enable_news_support'] = true;
+        }
+        $config['enableNewsSupport'] = $config['enable_news_support'];
```

Comment:
If $config['enableNewsSupport'] is false and $config['enable_news_support'] is not set, this option returns null

Reply:
Ne, in der config ist das default value für beides false. Null wäre aber auch nicht weiter wild, weil später explizit auf true getestet wird.

Reply:
Nagut.